build.properties

```diff
@@ -6,13 +6,13 @@ author = masa
 mod_file_name = malilib-fabric
 
 # Current mod version
-mod_version = 0.10.0-dev.21+arne.2
+mod_version = 0.10.0-dev.21+arne.3
```

Comment:
updated version number, cuz there are very very tiny changes to the build configuration